providers/src/airflow/providers/fab/www/extensions/init_views.py

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from flask import Flask
+
+log = logging.getLogger(__name__)
+
+
+def init_plugins(app):
+    """Integrate Flask and FAB with plugins."""
+    from airflow import plugins_manager
+
+    plugins_manager.initialize_flask_plugins()
+
+    appbuilder = app.appbuilder
+
+    for view in plugins_manager.flask_appbuilder_views:
```

Comment:
Somewhere in here we should issue a deprecation warning. In `init_plugins` is probably the right place. Fine with a follow up as long as we drop a TODO.

Reply:
Good point. I added one in the function `init_flask_plugins` in `airflow/api_fastapi/core_api/app.py`. I think it makes more sense. WDYT?